inc/saiswitch.h

```diff
@@ -976,8 +990,117 @@ typedef enum _sai_switch_attr_t
      * @default false
      */
     SAI_SWITCH_ATTR_SWITCH_SHELL_ENABLE,
+    
+    /**
+     * @brief Handle for switch profile id.
+     *
+     * Use this to retrive the Key-Vlaue pairs as part of switch 
+     * initialization. 
+     * @type sai_uint32_t
+     * @flags CREATE_ONLY
+     * @default 0
+     */
+    SAI_SWITCH_ATTR_SWITCH_PROFILE_ID,
+
+    /**
+     * @brief Device Information for switch initialization.
+     * 
+     * Hardware information format is based on SAI implementations by vendors.
+     * String is NULL terminated. Format is vendor specific. 
+     *   Example: Like PCI location, I2C adddress etc.
+     * In case of NULL, First NPU attached to CPU will be initialized.
+     * Single NPU case this attribute is optional.
+     *
+     * @type sai_s8_list_t
+     * @flags CREATE_ONLY
+     * @default empty
+     */
+    SAI_SWITCH_ATTR_SWITCH_HARDWARE_INFO,
+
+    /**
+     * @brief Vendor specific path name of the firmware to load.
+     * 
+     * @type sai_s8_list_t
+     * @flags CREATE_ONLY
+     * @default empty
+     */
+    SAI_SWITCH_ATTR_FIRMWARE_PATH_NAME,
+
+    /**
+     * @brief Set to switch initialization or connect to NPU/SDK.
+     *
+     * TRUE - Initialize switch/SDK. 
+     * FALSE - Connect to SDK. This will connects library to the initialized SDK.
+     * After this call the capability attributes should be ready for retrieval
+     * via sai_get_switch_attribute()
+     *
+     * @type bool
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_SWITCH_ATTR_INIT_SWITCH,
```

Comment:
should be the switch_id that it got from switch_init(). I see your point, in case the two host adapters need to communicate two different switch_id representing the same asic will be confusing. Then, I think we can still keep this switch_init(INIT) and let them return the same switch_id. So, we can still use all the switch attribute defined. Meanwhile, we add disconnect_switch(switch_id) function.